tests/core/_orchestrator/test_orchestrator__cancel_jobs.py

```diff
@@ -89,11 +91,10 @@ def test_cancel_job_with_subsequent_jobs_and_parallel_jobs():
     job3 = orchestrator._lock_dn_output_and_create_job(scenario.t3, "s_id", "e_id")
     job2bis = orchestrator._lock_dn_output_and_create_job(scenario.t2bis, "s_id", "e_id")
     job1.completed()
-
-    job2.running()
+    job2.pending()
```

Comment:
Why is this status changed?

Reply:
When a job was in the dispatched_processes dict, it was necessarily running. The test was wrong here since the job2 status should have already been pending.
Before the current PR, we tested if the job was in the dispatched_processes to know if we could cancel it (that's why the test passed). Now, we are testing if the job status is running instead.

Reply:
Ah I see, good catch. Thank you